storage/rocksdb/ha_rocksdb.cc

```diff
@@ -7952,13 +7961,13 @@ static int rocksdb_init_internal(void *const p) {
         rocksdb_use_hyper_clock_cache
             ? rocksdb::HyperClockCacheOptions(
                   rocksdb_block_cache_size, rocksdb_tbl_options->block_size,
-                  -1
-                  /* num_shard_bits */,
+                  rocksdb_block_cache_numshardbits /* num_shard_bits */,
```

Comment:
The variable name already tells it's numshardbits, making the comment redundant
```suggestion
                  rocksdb_block_cache_numshardbits,
```
This might also need a clang-format run to reflow (which may touch the rest of the file, so might be a good idea to limit it to this region)

Reply:
Updated the PR to fix this including running clang-format (phabricator also warned about that)